clients/java/src/main/java/io/camunda/zeebe/client/impl/command/ActivateJobsCommandImpl.java

```diff
@@ -46,6 +48,10 @@ public final class ActivateJobsCommandImpl
   private final Builder builder;
   private Duration requestTimeout;
 
+  private final Set<String> defaultTenantIds;
+  private final Set<String> customTenantIds;
+  private boolean areCustomTenantIdsSet;
```

Comment:
🔧 I don't think you need this flag. It can be replaced with `customTenantIds.isEmpty`